codebase2/codebase-sqlite/U/Codebase/Sqlite/Queries.hs

```diff
@@ -873,6 +876,18 @@ getCausalsWithoutBranchObjects = queryAtoms_ sql
 
 {- ORMOLU_ENABLE -}
 
+-- | Delete all hash objects of a given hash version.
+-- Leaves the corresponding `hash`es in the hash table alone.
+removeHashObjectsByHashingVersion :: DB m => HashVersion -> m ()
+removeHashObjectsByHashingVersion hashVersion =
+  execute sql (Only hashVersion)
+  where
+    sql =
+      [here|
+    DELETE FROM hash_object
+      WHERE hash_version = ?
+|]
+
```

Comment:
Weird formatting, lol.

Reply:
Yeah it's a bit weird that ormolu forces the `sql` and the `[here|` on different lines, but I don't personally find it to hard to read, it's such a short block 🤷🏼‍♂️